src/interaction.ts

```diff
@@ -191,14 +191,31 @@ export namespace interaction {
     export function warnNoRollback(this: void) {
         return window.showWarningMessage(localize('no rollback', "Nothing to rollback to."));
     }
-
+    
```

Comment:
This looks like it doesn't belong.

We should automate code formatting to avoid these in the future.